cves/kernel/CVE-2016-2782.yml

```diff
@@ -149,10 +147,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: "Looks like it was reported by a few engineers. They gave a description of the bug and provided informtion on the USB they used to replicate it on multiple systems."
```

Comment:
one guy found this, check [this](https://ubuntu.com/security/notices/USN-2929-1)

Reply:
I do mention him in this report, but in this answer specifically, I found a report from him that had a few other authors involved, so I didn't say he specifically found it. Here's that [report](https://www.exploit-db.com/exploits/39539). I'll add his name to this answer.